usr/bin/playtronos-systemreport

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# System Report
+#
+# Generates a system report archive which can contain useful information for
+# reporting bugs and troubleshooting.
+
+if [ $EUID -ne 0 ]; then
+  echo "$(basename "$0") must be run as root"
+  exit 1
+fi
+
+ENDCOLOR='\e[0m'
+RED='\e[0;31m'
+PURPLE='\e[0;35m'
+CYAN='\e[0;36m'
+
+TARGET_USER=playtron
+REPORT_NAME="report-$(date +%Y-%m-%d-%s)"
+REPORT_DIR_BASE="$(mktemp -d)"
+REPORT_DIR="${REPORT_DIR_BASE}/${REPORT_NAME}"
+mkdir -p "${REPORT_DIR}"
+
+echo -e "${CYAN}Generating system report...${ENDCOLOR}"
+
+# Hardware
+echo -e "  Collecting ${RED}hardware${ENDCOLOR} report..."
+dmesg >"${REPORT_DIR}/dmesg.log"
+lspci -tv -nn >"${REPORT_DIR}/lspci.out"
+lsusb -t -v >"${REPORT_DIR}/lsusb.out"
+cat /proc/bus/input/devices >"${REPORT_DIR}/input-devices.out"
+hwctl system-info >"${REPORT_DIR}/system.json" 2>/dev/null
+
+# Firmware
+fwupdmgr get-devices >"${REPORT_DIR}/firmware-devices.out" 2>&1
```

Comment:
For devices where the BIOS is not managed by `fwupdmgr`, there is no information about the BIOS version. That could be helpful to have.

```
echo "BIOS version: $(cat /sys/class/dmi/id/bios_version)" >>"${REPORT_DIR}/firmware-devices.out"
```